lambdas/mail_save.js

```diff
@@ -32,14 +32,21 @@ const handler = async(req, res) => {
           if (err) {
             send(res, statuses['bad request'], err.message);
           } else {
-            send(res, statuses['created'], 'The address was successfully subscribed');
+            send(
+              res,
+              statuses['created'],
+              'La suscripción se ha realizado correctamente'
+            );
           }
         }
       );
   }
 
-  return send(res, statuses['bad request'], 'Invalid email address');
-}
+  return send(res, statuses['bad request'], 'La dirección de email es inválida');
+};
 
-const cors = microCors({ allowMethods: ['POST'], origin: 'https://webconf.tech'});
+const cors = microCors({
+  allowMethods: ['POST'],
+  origin: ['https://webconf.tech', 'https://www.webconf.tech']
```

Comment:
Didn't we had a dev.webconf.tech also?

Reply:
Yeah, I'll add that when we deploy something :)